src/quo2/components/avatars/group_avatar.cljs

```diff
@@ -1,32 +1,47 @@
 (ns quo2.components.avatars.group-avatar
```

Comment:
perhaps we can move this to a view and style file while we are making the changes?